test/gen-wasm.py

```diff
@@ -43,6 +43,7 @@
     'f64': 0x7c,    # -4
     'v128': 0x7b,    # -5
     'funcref': 0x70,    # -0x10
+    'reference': 0x6b,    # -0x15
```

Comment:
Should we just use "ref" here like the spec does?

Reply:
The current wabt implementation uses this name and constant.
The spec has different names (ref/refnull) and constants for function references: 0x64 / 0x63

https://github.com/WebAssembly/function-references/blob/main/proposals/function-references/Overview.md#reference-types-2

The 0x6b also has a new name (`kStructRefCode`) according to v8 engine:
https://chromium.googlesource.com/v8/v8/+/refs/heads/main/src/wasm/wasm-constants.h#46

This whole naming needs to be changed, and #2562 does this, but this patch (should) precedes that.

Reply:
Fair enough lgtm as is then.